### ISSUE TITLE
[Enhancement] TouchEffect is lacking event for LongPress

### ISSUE THREAD

**Issue by Tommigun1980** [xamarin/XamarinCommunityToolkit#851]
### Steps to Reproduce

TouchEffect doesn't expose events as attachable properties (see https://github.com/xamarin/XamarinCommunityToolkit/issues/842#issuecomment-774565593), so one has to do:

```xaml
<ContentView>
    <ContentView.Effects>
        <xct:TouchEffect Completed="..."
```

However, there is no event for 'LongPress', so you can't do:

```xaml
<ContentView>
    <ContentView.Effects>
        <xct:TouchEffect LongPress="..."
```

This results in a situation where long presses can only be handled via a Command, and there doesn't seem to be any way of accessing it as an event (which is imperative when implementing controls etc).

I am filing this as a bug as it seems the API is not complete.

Thank you.

### Expected Behavior

A long press event should exist.

### Actual Behavior

No long press event exists.

### Basic Information

- Version with issue: 1.0.2
- Last known good version: None?
- IDE:
- Platform Target Frameworks: 
  - iOS:  14.4

**Comment by jfversluis** [comment]
It a bug, everything works as intended and this is something wished to be added.

This is probably useful for a lot of people

**Comment by AndreiMisiukevich** [comment]
Fixed in #1064
